src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4599,6 +4599,12 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="IDS_AwaitInCatchAndFinally" xml:space="preserve">
     <value>await in catch blocks and finally blocks</value>
   </data>
+  <data name="IDS_FeatureBinaryLiteral" xml:space="preserve">
+    <value>binary literals</value>
+  </data>
+  <data name="IDS_FeatureDigitSeparator" xml:space="preserve">
+    <value>numerics with underscores</value>
```

Comment:
But it should at least be consistent between C# and VB, which currently uses `digit separators`. Actually, I like `digit separators`.